.github/workflows/update-viewer.yml

```diff
@@ -0,0 +1,28 @@
+name: Update viewer.php - viewer.html got updated so viewer.php needs an update
+
+on:
+  pull_request:
+    types: [opened, ready_for_review, reopened, synchronize]
+    paths:
+      - 'js/pdfjs/web/viewer.html'
+
+permissions:
+  contents: read
+
+concurrency:
+  group: update-viewer-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  update-viewer:
```

Comment:
Per above, it might be better to name it `check-viewer`.

Reply:
done